jobs/initial_data/__init__.py

```diff
@@ -17,6 +22,17 @@ class Meta:
         commit_default = False
         design_file = "designs/0001_design.yaml.j2"
         context_class = InitialDesignContext
+        version = "1.0.0"
+        description = "Establish the devices and site information for four sites: IAD5, LGA1, LAX11, SEA11."
+        docs = """This design creates the following objects in the source of truth to establish the initia network environment in  four sites: IAD5, LGA1, LAX11, SEA11.
```

Comment:
I think this has the potential to confuse people since the `docs` feature is part of the design lifecycle project we're still working on.